Tests/LLVMTests/FileCheck.swift

```diff
@@ -416,7 +431,7 @@ private func check(input b : String, against checkStrings : [CheckString]) -> Bo
   var failedChecks = false
 
   // This holds all the current filecheck variables.
-  var variableTable = [String:String]()
+  var variableTable = BoxedTable()
```

Comment:
I considered it, then realized I only wrote this thing because the C++ just throws a reference to this dictionary everywhere so I only need reference semantics until I can find a way to remove the abstraction altogether.